client/components/dashboard/dashboard-directive.html

```diff
@@ -47,7 +47,7 @@
           <div class="pk-widget-content" flex layout="column"
                ng-style="{height: isWidgetScrollable(widget, container) ? container.model.container.height + 'px' : 'auto',
                           'min-height': container.model.container.height + 'px',
-                          'overflow-y': isWidgetScrollable(widget, container) ? 'auto' : 'visible'}">
+                          'overflow-y': isWidgetScrollable(widget, container) ? 'auto' : 'hidden'}">
```

Comment:
There isn't actually cropping, because the chart is explicitly sized to the container's dimensions in the gvizWidgetDirective.  There's a 1-pixel margin bleedover that is indistinguishable for the user.  On Linux, this causes a vertical scrollbar to appear and take up another ~16 pixels of width, which in turn causes a horizontal scrollbar to appear and take up ~16 pixels of height.